File: ios/Classes/CallMode.swift
import Foundation

enum CallMode {
    case none
    case inProgress
    case ringing
    
    func toString() -> String {
        switch self {
        case .none:
            return  "none"
        case .inProgress:
            return "inProgress"
        case .ringing:
            return "ringing"
        }
     }
}

File: android/src/main/kotlin/com/kardenvan/call_mode/CallMode.kt
package com.kardenvan.call_mode

enum class CallMode {
    RINGING,
    NONE,
    IN_PROGRESS;

    
    override fun toString(): String {
        return when (this) {
            RINGING -> "ringing"
            NONE -> "none"
            IN_PROGRESS -> "inProgress"
        }
    }
}